src/navigator/Navigator.tsx

```diff
@@ -511,6 +513,12 @@ const settingsScreens = (Navigator: typeof Stack) => (
       name={Screens.FiatConnectLinkAccount}
       component={FiatConnectLinkAccountScreen}
     />
+    <Navigator.Screen options={headerWithBackButton} name={Screens.Persona} component={Persona} />
```

Comment:
I don't think the Persona component is a screen. It's currently just a button, which on clicking opens the persona flow. Is the plan to refactor this component to just load Persona flow on load? Or do we just want to refactor the persona stuff into it's own saga and drop that component?

Reply:
Added a new `KycLanding` screen instead!